sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -135,116 +61,34 @@ The authentication events trigger output binding allows a function to send authe
 
 ## Documentation
 
-* One the function has been published, there's some good reading about logging and metrics that can be found [here](https://learn.microsoft.com/azure/azure-functions/functions-monitor-log-analytics?tabs=csharp)
-
-* For API Documentation, please see the (Link TBD)
+* Once the function has been published, there's some good reading about logging and metrics that can be found [here](https://learn.microsoft.com/azure/azure-functions/functions-monitor-log-analytics?tabs=csharp)
 * Once this moves to preview, we except no breaking changes and would be as simple as removing the the NuGet source that points to the private preview.
```

Comment:
delete this

Reply:
Moved `* Once the function has been published, there's some good reading about logging and metrics that can be found [here](https://learn.microsoft.com/azure/azure-functions/functions-monitor-log-analytics?tabs=csharp)` link below in next steps section